### PR TITLE
#911 new telegram version

### LINKED ISSUE #911: Support new Telegram version 8.4
Database schema has changed, but the lib used by us as GPL plugin was updated recently: https://github.com/DrKLO/Telegram.

### PR DESCRIPTION
**Pull request by hauck-jvsh**
This closes #911.

### CODE REVIEW

**Comment by lfcnassif** [comment]
Thanks @hauck-jvsh! I will try to take a look tomorrow. Closes #911

File: iped-parsers/iped-parsers-impl/src/main/java/dpf/ap/gpinf/telegramextractor/MapTypeMSG.java [hunk 1]
```diff
@@ -41,7 +41,7 @@
 
         msg.put(6, Messages.getString("TelegramReport.pinnMessage"));
         msg.put(7, Messages.getString("TelegramReport.UserJoinLink"));
-        msg.put(8, Messages.getString("TelegramReport.ChangeToGroup"));
+        msg.put(8, Messages.getString("TelegramReport.ChannelMigratedFromGroup"));
```
Comment (lfcnassif) [inline iped-parsers/iped-parsers-impl/src/main/java/dpf/ap/gpinf/telegramextractor/MapTypeMSG.java:44]: Thanks @hauck-jvsh! One question: are these mapping changes correct for old telegram versions, right?

Comment (hauck-jvsh) [reply iped-parsers/iped-parsers-impl/src/main/java/dpf/ap/gpinf/telegramextractor/MapTypeMSG.java:44]: Yes, I will take another look just to be sure and put the reference. I think that it was wrong in the past.

Comment (hauck-jvsh) [reply iped-parsers/iped-parsers-impl/src/main/java/dpf/ap/gpinf/telegramextractor/MapTypeMSG.java:44]: This the reference
```
 case 8:
      self = .channelMigratedFromGroup(title: decoder.decodeStringForKey("title", orElse: ""), groupId: PeerId(decoder.decodeInt64ForKey("groupId", orElse: 0)))
```
The file can be found in:
https://github.com/TelegramMessenger/Telegram-iOS/blob/master/submodules/TelegramCore/Sources/SyncCore/SyncCore_TelegramMediaAction.swift

Comment (lfcnassif) [reply iped-parsers/iped-parsers-impl/src/main/java/dpf/ap/gpinf/telegramextractor/MapTypeMSG.java:49]: Thanks for double checking @hauck-jvsh

**Pull request merged**